lms/config.py

```diff
@@ -99,6 +99,8 @@ def __post_init__(self):
     _Setting("blackboard_api_client_secret"),
     _Setting("jstor_api_url"),
     _Setting("jstor_api_secret"),
+    _Setting("youtube_api_url"),
```

Comment:
youtube_api_url might not change that often and probably never across different environments (ie one for QA other for prod).

We could just use a constant at the service level YOUTUBE_API_URL = `https://` with the hard coded value.

Not that the env vars are bad but it's  currently a bit of pain to set them in US/CA/QA